curl/curl_d.php

```diff
@@ -48,189 +60,820 @@
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_READFUNCTION', 20012);
+/**
+ * A callback accepting two parameters. The first is the cURL resource, the second is a string with the header data to be written.
+ * The header data must be written by this callback. Return the number of bytes written.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_HEADERFUNCTION', 20079);
+/**
+ * The maximum amount of HTTP redirections to follow. Use this option alongside <b>CURLOPT_FOLLOWLOCATION</b>.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_MAXREDIRS', 68);
+/**
+ * The maximum amount of persistent connections that are allowed.
+ * When the limit is reached, <b>CURLOPT_CLOSEPOLICY</b> is used to determine which connection to close.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_MAXCONNECTS', 71);
+/**
+ * This option is deprecated, as it was never implemented in cURL and never had any effect.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ * @deprecated 5.6
+ */
 define ('CURLOPT_CLOSEPOLICY', 72);
+/**
+ * <b>TRUE</b> to force the use of a new connection instead of a cached one.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_FRESH_CONNECT', 74);
+/**
+ * <b>TRUE</b> to force the connection to explicitly close when it has finished processing, and not be pooled for reuse.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_FORBID_REUSE', 75);
+/**
+ * A filename to be used to seed the random number generator for SSL.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_RANDOM_FILE', 10076);
+/**
+ * Like <b>CURLOPT_RANDOM_FILE</b>, except a filename to an Entropy Gathering Daemon socket.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_EGDSOCKET', 10077);
+
+/**
+ * The number of seconds to wait while trying to connect. Use 0 to wait indefinitely.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ */
 define ('CURLOPT_CONNECTTIMEOUT', 78);
+
+/**
+ * The number of milliseconds to wait while trying to connect. Use 0 to wait indefinitely.
+ * If libcurl is built to use the standard system name resolver, that portion of the connect
+ * will still use full-second resolution for timeouts with a minimum timeout allowed of one second.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ * @since 5.2.3
+ */
 define ('CURLOPT_CONNECTTIMEOUT_MS', 156);
+/**
+ * <b>FALSE</b> to stop cURL from verifying the peer's certificate.
+ * Alternate certificates to verify against can be specified with the <b>CURLOPT_CAINFO</b> option or
+ * a certificate directory can be specified with the <b>CURLOPT_CAPATH</b> option.
+ * @link https://www.php.net/manual/en/function.curl-setopt.php
+ * @since 7.1
```

Comment:
Hi!

This is incorrect. The option of "CURLOPT_SSL_VERIFYPEER" is available since at least PHP 5.
The documentation says:

https://www.php.net/manual/en/function.curl-setopt.php
"TRUE by default as of **cURL 7.10**. Default bundle installed as of **cURL 7.10**."

It's referring to "cURL" package, not PHP.
This error generate a fake error in PHPStorm.

![image](https://user-images.githubusercontent.com/4594902/84355403-d6902f00-abc2-11ea-88a6-9d6a84346859.png)